carma-messenger-core/j2735_convertor/src/control_message_convertor.cpp

```diff
@@ -25,79 +25,6 @@ namespace geofence_control
 // Convert j2735_msgs to cav_msgs
 ////
 
-void convert(const j2735_msgs::ControlMessage& in_msg, cav_msgs::ControlMessage& out_msg)
```

Comment:
I assume these removed blocks are for the now removed message specs?

Reply:
yeah all removed ones are old spec.